espflash/src/cli/mod.rs

```diff
@@ -121,6 +124,65 @@ pub fn save_elf_as_image(
         }
     }
 
+    // merge_bin is TRUE
+    // merge bootloader, partition table and app binaries
+    // basic functionality, only merge 3 binaries
+    if merge {
+        use std::io::Write;
```

Comment:
not sure we use this import style anywhere in this repo? maybe just move it to the top?